test-e2e/test_calico_networking.py

```diff
@@ -117,7 +117,8 @@ def test_calico_disabled(docker_backend: Docker, artifact_path: Path,
         )
 
         calico_units = ["dcos-calico-felix", "dcos-calico-bird",
-                        "dcos-calico-confd", "dcos-calico-libnetwork-plugin"]
+                        "dcos-calico-confd", "dcos-calico-libnetwork-plugin",
+                        "dcos-etcd"]
```

Comment:
We should add `!packages/etcd/**` to the conditional running of this test so that undoing the change to `packages/etcd/extra/systemd/dcos-etcd.service` would fail.

Reply:
:+1:

Reply:
Good call, done and repushed.